src/test/java/net/sf/xsshtmlfilter/HTMLFilterTest.java

```diff
@@ -1,16 +1,15 @@
 package net.sf.xsshtmlfilter;
 
-import net.sf.xsshtmlfilter.HTMLFilter;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
```

Comment:
you can't be enforcing a private and unbeknown to all others code style.

in lei of any documented code style for this project, and any precedence in its other java files, i would simply avoiding changes in pull requests that are unnecessary. that's a pretty normal expectation in the open source world.